narr_mod/three_act.py

```
@@ -1,190 +1,463 @@
+    number: int
+    type: ActType
+    name: str
+    name_en: str
+    description: str
+    description_en: str
+    color: str
+    elements: List[ActElement]
+    expected_length: float  # процент от общей длины
+
+@dataclass
+class AnalysisResult:
```

Ruff (0.8.2)

Redefinition of unused `AnalysisResult` from line 6

(F811)